src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -496,8 +497,15 @@ private static InterfaceDeclarationSyntax GenerateImplementationInterface(ComInt
                         .OfType<GeneratedStubCodeContext>()
                         .Select(ctx => ctx.Stub.Node)
                         .Concat(shadowImplementations)
-                        .Concat(inheritedStubs)))
-                .AddAttributeLists(AttributeList(SingletonSeparatedList(Attribute(NameSyntaxes.System_Runtime_InteropServices_DynamicInterfaceCastableImplementationAttribute))));
+                        .Concat(inheritedStubs)));
+
+            // If the user has specified 'ManagedObjectWrapper', it means that the COM interface will never be used to marshal a native
```

Comment:
Are there warnings if only the opposite direction is specified (that is, `ComInterfaceOptions.ComObjectWrapper`)?

Reply:
How was this resolved? Was it verified it is clean? Confirmation would be appreciated.

Reply:
Oh sorry, I thought you had followed the chat offline on Teams 😅
I completely reverted this change, so the PR is not touching this part at all anymore.